CMakeLists.txt

```diff
@@ -55,7 +55,8 @@ endif()
 # to configure the options specify them in the command line or change them in the cmake UI.
 # Don't edit the makefile!
 option(BUILD_SHARED_LIBS "Build shared library" OFF)
```

Comment:
I guess change this to CAPSTONE_BUILD_SHARED_LIBS is better, how do you think @Rot127 ?